docs/CONTENTS.md

```diff
@@ -47,7 +47,7 @@ Major directories and files in the source distribution:
         - environments\
             - Anaconda environment files for supported developer platforms
     - Windows-only
-        - bootstrap.bat (see [Windows](README_WINDOWS.md))
```

Comment:
Should we merge BUILD_WIN10 and _WIN11 into a file by this name?

Reply:
We should just link to one of them. Originally Win10 was the main build instructions and Win11 [just redirected to Win10](https://github.com/WasatchPhotonics/ENLIGHTEN/wiki/Build-Win11).

But recently I was able to make a build from Windows 11 using almost the exact same steps. (Slight tweak to Bootstrap.bat #344)